### PR TITLE
Power safety

### PR DESCRIPTION
**Pull request by Vertiq-Ben**
- Added documentation for power_safety for 2306 speed

### CODE REVIEW

File: docs/clients/power_safety.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+| 2      | fault_latching        | get, set, save | uint8     |       | Determines if the motor is latching. When latching, if there are any bits set in fault_ever, motor will stay in Safe Mode until they are cleared. When not latching, motor will enter and exit Safe Mode only on current faults based on fault_now |
++--------+-----------------------+----------------+-----------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 3      | volt_input_low        | get, set, save | float     | fix16 | !                                                                                                                                                                                                                                                  |
```
Comment (iq-fred) [inline docs/clients/power_safety.rst:124]: Should probably add descriptions to these guys

**Review by iq-fred — CHANGES_REQUESTED** [review]

File: docs/clients/power_safety.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+| 2      | fault_latching        | get, set, save | uint8     |       | Determines if the motor is latching. When latching, if there are any bits set in fault_ever, motor will stay in Safe Mode until they are cleared. When not latching, motor will enter and exit Safe Mode only on current faults based on fault_now |
++--------+-----------------------+----------------+-----------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 3      | volt_input_low        | get, set, save | float     | fix16 | Minimum threshold value for voltage                                                                                                                                                                                                                |
```
Comment (iq-matt) [inline docs/clients/power_safety.rst:124]: Incorrect units.  Match to new python Unit.

**Review by iq-matt — CHANGES_REQUESTED** [review]
Incorrect units

File: docs/clients/power_safety.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
+
+Power Safety
+-------------
+
+The Power Safety protects the motor by checking various parameters in the motor, such as voltage and current, to make sure the values are within a specific range. If the values are above or below the thresholds, the motor will coast and locked down to prevent any commands from being processed.
```
Comment (iq-fred) [inline docs/clients/power_safety.rst:5]: "lock down" not "locked down". Also, probably worth nothing that it "unlocks" once the parameters are back in the safe region

**Review by iq-fred — CHANGES_REQUESTED** [review]

**Review by iq-matt — APPROVED** [review]

**Review by iq-fred — APPROVED** [review]

**Pull request merged**